### ISSUE TITLE
Model not properly displayed in IJulia

### ISSUE THREAD

**Issue by ShuhuaGao** [jump-dev/JuMP.jl#2328]
When we do not append a semicolon after an expression, it will be displayed automatically in both REPL and IJulia (i.e., Jupyter notebook). At the beginning of an optimization program, we usually define a `model`. Such a model can be properly displayed _automatically_ in REPL but not in IJulia.

## A minimal reproducible example in IJulia
![Snipaste_2020-09-11_20-50-50](https://user-images.githubusercontent.com/20141984/92927624-84380600-f470-11ea-8a75-5e25a305441f.png)

As we see above, the `show` and the `display("text/plain", model)` methods work well. However, the single argument `display` only shows incomplete information. Besides, the automatic display at the end of a cell does not work as expected. 

In this [discussion](https://discourse.julialang.org/t/what-function-display-does-ijulia-call-at-the-end-of-each-cell/33554/2?u=shuhua) as well as the [official documentation](https://docs.julialang.org/en/v1/base/io-network/#Base.Multimedia.display), it is recommended to override `show` to customize the behaviour of `display`.  It is strange that `show` works well as above in IJulia but not `display`. 

This issue may be related to issue #2293.

## Version info
```julia
julia> versioninfo()
Julia Version 1.5.0
Commit 96786e22cc (2020-08-01 23:44 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i7-3770 CPU @ 3.40GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-9.0.1 (ORCJIT, ivybridge)

(@v1.5) pkg> st IJulia
Status `C:\Users\Gao Shuhua\.julia\environments\v1.5\Project.toml`
  [7073ff75] IJulia v1.21.2

(@v1.5) pkg> st JuMP
Status `C:\Users\Gao Shuhua\.julia\environments\v1.5\Project.toml`
  [4076af6c] JuMP v0.21.3
```

**Comment by blegat** [comment]
> Besides, the automatic display at the end of a cell does not work as expected.

What do you mean by this?

In IJulia, the mime is `text/latex` so we show the model in LaTeX. Here there is no objective and no constraint so the output you see is expected

**Comment by ShuhuaGao** [comment]
Thanks for your quick response. 

> What do you mean by this?

I mean that we simply run a cell in Jupyter notebook without calling a display function like `print`, `show`, or `display` explicitly. By default, IJulia will display the final result of the cell *automatically* (similar to REPL) if we did not terminate it with a semicolon.

I figured it out according to your reply. Here, it seems IJulia tries to display the `model` using `display("text/latex", model)`. However, in the beginning, the *feasibility* and the *subject to* are both empty, and thus the display shown above is actually right. I thought that the display in IJulia was identical to that in REPL, which was my mistake. To get the same display like that in REPL, we should call `display("text/plain", model)` or `show(model)` manually, as demonstrated in the above figure.

Once we have optimized the model, the default display in IJulia is obviously right now.

![Snipaste_2020-09-12_11-05-38](https://user-images.githubusercontent.com/20141984/92985732-e849e100-f4e7-11ea-8144-ccaac9aea890.png)
